app/assets/stylesheets/custom.scss

```diff
@@ -22,8 +22,33 @@ input {
 }
 
 
-.log-wrapper{
-  margin-left: auto;
+.log-wrap{
   margin-right: 0;
+  margin-left: auto;
+  padding-right:0;
+  width: 700px;
```

Comment:
why don't you use bootstrap's grid? 

Container + columns, instead of hardcoding the width. This will make responsiveness harder to achieve